evennia/contrib/events/README.md

```diff
@@ -0,0 +1,646 @@
+
+#### Permissions in settings
+
+The events contrib adds three [permissions](https://github.com/evennia/evennia/wiki/Locks#permissions) in the settings.  You can override them by changing the settings into your `server/conf/settings.py` file (see below for an example).  The settings defined in the events contrib are:
+
+- `EVENTS_WITH_VALIDATION`: this defines a permission that can edit events, but will need approval.  If you set this to `"wizards"`, for instance, users with the permission `"wizards"` will be able to edit events.  These events will not be connected, though, and will need to be checked and approved by an administrator.  This setting can contain `None`, meaning that no user is allowed to edit events with validation.
+- `EVENTS_WITHOUT_VALIDATION`: this setting defines a permission allowing editing of events without needing validation.  By default, this setting is set to `"immortals"`.  It means that immortals can edit events, and they will be connected when they leave the editor, without needing approval.
+- `EVENTS_VALIDATING`: this last setting defines who can validate events.  By default, this is set to `"immortals"`, meaning only immortals can see events needing validation, accept or reject them.
+
+You can override all these settings in your `server/conf/settings.py` file.  For instance:
+
+```python
+# ... other settings ...
+
+# Event settings
+EVENTS_WITH_VALIDATION = "wizards"
+EVENTS_WITHOUT_VALIDATION = "immortals"
+EVENTS_VALIDATING = "immortals"
+```
+
+This set of settings means that:
+
+1. Wizards can edit events, but they will need to be individually approved before they are connected.  Wizards will be able to add whatever they want, but before their code runs, it will have to be checked and approved by an immortal.
+2. Immortals can edit events, their work doesn't need to be approved.  It is automatically accepted and connected.
+3. Immortals can also see events that need approval (these produced by wizards) and accept or reject them.  Whenever accepted, the event is connected and will fire without constraint whenever it has to.
+
+In addition, there is another setting that must be set if you plan on using the time-related events (events that are scheduled at specific, in-game times).  You would need to specify the type of calendar you are using.  By default, time-related events are disabled.  You can change the `EVENTS_CALENDAR` to set it to:
+
+- `"standard"`: the standard calendar, with standard days, months, years and so on.
+- `"custom"`: a custom calendar that will use the [custom_gametime](https://github.com/evennia/evennia/blob/master/evennia/contrib/custom_gametime.py) contrib to schedule events.
+- A special callback to schedule time-related events in a way not supported by the `gametime` utility and the `custom_gametime` contrib (see below).
+
+#### Permissions on individual users
```

Comment:
This could just be merged with the previous section and a reference given to the permission section of the wiki.